npm/cypress-schematic/src/ct.spec.ts

```diff
@@ -24,7 +24,7 @@ const copyAngularMount = async (projectPath: string) => {
 
 const cypressSchematicPackagePath = path.join(__dirname, '..')
 
-const ANGULAR_PROJECTS: ProjectFixtureDir[] = ['angular-20', 'angular-21']
+const ANGULAR_PROJECTS: ProjectFixtureDir[] = ['angular-19', 'angular-20']
```

Comment:
we have to downgrade the test because the `zoneless` handler isn't supported within the binary yet (see the PR description)